notifications/2024-01-07_upgrades.md

```diff
@@ -0,0 +1,67 @@
+**What is happening?**
+
+CHG0055023 - MCS SILVER - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade
+
+CHG0055017 - MCS SILVER - 2024 Q1 Patching - Upgrade to OCP 4.13.24
+
+CHG0055024 - MCS GOLD - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade
+
+CHG0055018 - MCS GOLD - 2024 Q1 Patching - Upgrade to OCP 4.13.24
+
+CHG0055022 - MCS GOLDDR - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade
+
+CHG0055020 - MCS GOLDDR - 2024 Q1 Patching - Upgrade to OCP 4.13.24
+
+These six changes represent the work involved to upgrade the production clusters from Openshift 4.12 to 4.13.
+
+**When?**
+
+CHG0055023 - MCS SILVER - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade will commence at 0600 on Sunday January 7 and run as late as 1000.
```

Comment:
```suggestion
CHG0055023 - MCS SILVER - 2024 Q1 Patching - Hard Prune of Image registry, ETCD Defrag and F5 Upgrade will commence at 6:00AM on Sunday January 7 and run as late as 10:00AM.
```

Reply:
Fixed in all the 3 sections. I am so used to using military time but agree that either we use the AM/PM style or the military style, so will go with AM/PM type.

Reply:
I think the customers find AM/PM clearer https://chat.developer.gov.bc.ca/group/internal-platform-services?msg=ubx7Jw3XNp5TjQmnY